protocol/development-workflow/README.md

```diff
@@ -142,3 +142,30 @@ push the new `master` to the staging app.
 Further reading:
 [A Successful Git Branching Model](http://nvie.com/posts/a-successful-git-branching-model/)
 [Continuous Delivery (Wikipedia)](https://en.wikipedia.org/wiki/Continuous_delivery)
+
+
+Small, discretionary releases
+--------------------------
+Sometimes a change is so small that it doesn't warrent the entire 
+workflow process described above. For instance, a single typo in
```

Comment:
I would add the examples that I think are even more classic: fixing/refactoring tests